src/test/java/org/aksw/gerbil/dataset/impl/ritter/RitterDatasetTest.java

```diff
@@ -47,17 +47,17 @@ public InitializableDataset createDataset(File file) {
     public static Collection<Object[]> data() {
         List<Object[]> testConfigs = new ArrayList<Object[]>();
         testConfigs.add(new Object[] {
-                "@paulwalk	O\nIt	O\n's	O\nthe	O\nview	O\nfrom	O\nwhere	O\nI	O\n'm	O\nliving	O\nfor	O\ntwo	O\nweeks	O\n.	O\nEmpire	B-facility\nState	I-facility\nBuilding	I-facility\n=	O\nESB	B-facility\n.	O\nPretty	O\nbad	O\nstorm	O\nhere	O\nlast	O\nevening	O\n.	O\n	\nFrom	O\nGreen	O\nNewsfeed	O\n:	O\nAHFA	B-other\nextends	O\ndeadline	O\nfor	O\nSage	B-other\nAward	I-other\nto	O\nNov	O\n.	O\n5	O\nhttp://tinyurl.com/24agj38	O",
```

Comment:
You changed the input but I think we got this input from the dataset, or? If this is the case, we should stick to the original data, even if we do not process the apostrophe correctly at the moment.

Reply:
I adapted the test to the original text.